src/core/tsi/ssl_transport_security_utils.h

```diff
@@ -142,6 +144,22 @@ tsi_result SslProtectorUnprotect(const unsigned char* protected_frames_bytes,
                                  unsigned char* unprotected_bytes,
                                  size_t* unprotected_bytes_size);
 
+// Verifies that `crl` was signed by `issuer`
```

Comment:
micro-nit: Please add periods at the end of the function comments.

Reply:
Done

Reply:
I believe this has been resolved for a time, please double check